views/components/ColorPicker.jsx

```diff
@@ -18,22 +13,49 @@ const ColorPicker = ({
 
   return (
     <div className={classes.swatchContainer}>
-      <Typography variant='caption' className={classes.colorLabel}>
-          Color
+      <Typography variant="caption" className={classes.colorLabel}>
+        Color
       </Typography>
       <Paper
         className={classes.swatch}
         style={{ backgroundColor: color }}
         onClick={() => setColorPickerToggle(true)}
       />
-    {isColorPickerOpen && (
-      <div className={classes.popover} ref={popover}>
-        <HexColorPicker 
-          color={color} 
-          onChange={(newColor) => onColorChange({target: { name: colorKey, value: newColor }}, idx)} 
-        />
-      </div>
-    )}
+      {isColorPickerOpen && (
+        <div
+          className={classes.popover}
+          ref={popover}
+          key={idx + 'color_picker'}
```

Comment:
do we need a key prop here?

Reply:
when there are additional values added to the form there's a warning and this key removes that warning

Reply:
https://github.com/AMP-SCZ/dpdash/pull/183/commits/093968d3f77a37a35aabae4a2e6ea7f246920945